waku/waku_lightpush/protocol.nim

```diff
@@ -79,7 +80,7 @@ proc handleRequest*(
 
 proc initProtocolHandler(wl: WakuLightPush) =
   proc handle(conn: Connection, proto: string) {.async.} =
-    let buffer = await conn.readLp(MaxRpcSize.int)
+    let buffer = await conn.readLp(wl.maxRpcSize)
```

Comment:
I may be missing something, but why are we not using the `-1` default here too?

Reply:
i started with changing lightpush this way..and later realised -1 is simpler approach. Will change for lightpush as well, as this much complexity is not needed.